pkg/transform/snowplow_gtmss_preview.go

```diff
@@ -11,14 +13,16 @@ import (
 
 // GTMSSPreviewConfig is a configuration object for the spEnrichedToJson transformation
 type GTMSSPreviewConfig struct {
+	Timeout int `hcl:"timeout_seconds,optional"`
```

Comment:
since 'timeout' is used for something with different semantics in other transformations, could we name this one something else? 'expiry' perhaps

Reply:
Nice, I like `expiry`